src/pages/zh-cn/reference/directives-reference.md

```diff
@@ -0,0 +1,242 @@
+<h1 set:html={rawHTMLString} />
+  <!-- 输出：<h1>Hello <strong>World</strong></h1> -->
+```
+
+你也可以在 `<Fragment>` 上使用 `set:html` 来避免添加不必要的包装元素。这在从 CMS 获取 HTML 时特别有用。
+
+```astro
+---
+const cmsContent = await fetchHTMLFromMyCMS();
+---
+<Fragment set:html={cmsContent}>
+```
+
+### `set:text`
+
+`set:text={string}` 将文本字符串注入元素中，类似于设置 `el.innerText`。与 `set:html` 不同的是，传递的 `string` 值会被 Astro 自动转义。
+
+这相当于直接将变量传入模板表达式（例如：`<div>{someText}</div>`），因此这个指令并不不常用。
+
+## 客户端指令
+
+这些指令描述了如何激活 [UI 框架组件](/zh-cn/core-concepts/framework-components/)。
+
+默认情况下，UI 框架组件不会在客户端激活。如果没有 `client:*` 指令，它的 HTML 将被渲染到页面上，而无需 JavaScript。
+
+### `client:load`
+
+- **优先级**：高
+- **适用于**：立即可见的UI元素，需要尽快进行互动。
+
+在页面加载时，立即加载并激活组件的 JavaScript。
+
+```astro
+<BuyButton client:load />
+```
+
+### `client:idle`
+
+- **优先级**：中
+- **适用于**：优先级较低的 UI 元素，不需要立即进行互动。
+
+一旦页面完成了初始加载，并触发 `requestIdleCallback` 事件，就会加载并激活组件中的 JavaScript。如果你所在的浏览器不支持 [`requestIdleCallback`](https://developer.mozilla.org/en-US/docs/Web/API/Window/requestIdleCallback)，那么就会使用文档 [`load`](https://developer.mozilla.org/en-US/docs/Web/API/Window/load_event) 事件。
+
+```astro
+<ShowHideButton client:idle />
+```
+
+### `client:visible`
+
+- **优先级**：低
+- **适用于**：低优先级 UI 元素，这些元素要么在页面的下方（折叠中），要么加载时非常耗费资源，如果用户没有看到这些元素，你宁愿不加载它们。
+
+一旦组件进入用户的视口，就加载组件的 JavaScript 并使其激活。这在其内部使用 `IntersectionObserver` 来跟踪无障碍。
+
+```astro
+<HeavyImageCarousel client:visible />
+```
+
+### `client:media`
+
+- **优先级**：低
+- **适用于**：侧边栏折叠或其他可能只在某些屏幕尺寸上可见的元素。
+
+`client:media={string}` 一旦满足一定的 CSS 媒体查询条件，就会加载并激活组件的 JavaScript。
+
+:::note
+如果该组件在 CSS 中已隐藏，只在满足媒体查询条件时显示，那么只需使用 `client:visible` 就行，而不必传递相同的媒体查询给指令。
+:::
+
+```astro
+<SidebarToggle client:media="(max-width: 50em)" />
+```
+
+### `client:only`
+
+`client:only={string}` **跳过** HTML 服务端渲染，只在客户端进行渲染。它的作用类似于 `client:load`，它在页面加载时立即加载、渲染和润色组件。
+
+**你必须正确传递组件所用框架！**因为 Astro 不会在构建过程中/在服务器上运行该组件，Astro 不知道你的组件使用什么框架，除非你明确告诉它。
+
+```astro
+<SomeReactComponent client:only="react" />
+<SomePreactComponent client:only="preact" />
+<SomeSvelteComponent client:only="svelte" />
+<SomeVueComponent client:only="vue" />
+<SomeSolidComponent client:only="solid-js" />
+```
+
+## 脚本和样式指令
+
+这些指令只能用于 HTML 的 `<script>` 和 `<style>` 标签，以控制客户端 JavaScript 和 CSS 在页面上的处理方式。
+
+### `is:global`
+
+默认情况下，Astro 会自动将 `<style>` CSS 规则扩展到组件上。你可以用 `is:global` 指令禁用该行为。
+
+`is:global` 使 `<style>` 标签的内容在包含该组件的页面上全面应用。这使得 Astro 的 CSS 作用域系统失效。这相当于用 `:global()` 来包装 `<style>` 标签内的所有选择器。
+
+你可以在组件中同时使用 `<style>` 和 `style is:global>`，创建一些全局样式规则，同时仍对大部分组件 CSS 进行作用域控制。
+
+📚 有关全局样式工作的更多细节，请参见[样式 & CSS](/zh-cn/guides/styling/#全局样式)页面。
+
+```astro
+<style is:global>
+  body a { color: red; }
+</style>
+```
+
+### `is:inline`
+
+默认情况下，Astro 会处理、压缩和捆绑它在页面上看到的任何 `<script>` 和 `<style>` 标签。你可以用 `is:inline` 指令禁用该行为。
+
+`is:inline` 可以让 Astro 将 `<script>` 或 `<style>` 标签原封不动地留在最终输出的 HTML 中。这些内容将不会被处理、压缩或捆绑。它限制了 Astro 的一些功能，比如导入 npm 包或使用像 Sass 这样的 CSS 编译语言。
+
+`is:inline` 指令意味着 `<style>` 和 `<script>` 标签。
```

Comment:
This is the start of a list, I'm not sure if this is different in Chinese, but I think here should be used "：" instead of "。":
```suggestion
`is:inline` 指令意味着 `<style>` 和 `<script>` 标签：
```